From 4059161333a14a0c705efecc5765b6cb2a29ae02 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Oct 22 2018 18:43:14 +0000
Subject: Makefile: add missing DESTDIR variable use


Patch borrowed from OpenEmbedded, available at
recipes/libaio/libaio-0.3.106/destdir.patch in their source tree.

It just adds support for the traditional DESTDIR variable to install
the library in a different sysroot than the normal /.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
[JEM: I couldn't verify the initial author of this change, but it's
 trivial.  For information on the DESTDIR convention, see:
 https://www.gnu.org/prep/standards/html_node/DESTDIR.html]
Signed-off-by: Jeff Moyer <jmoyer@redhat.com>

---

diff --git a/Makefile b/Makefile
index e9dbdb0..c1fb831 100644
--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ all:
 	@$(MAKE) -C src
 
 install:
-	@$(MAKE) -C src install prefix=$(prefix) includedir=$(includedir) libdir=$(libdir)
+	@$(MAKE) -C src install prefix=$(DESTDIR)$(prefix) includedir=$(DESTDIR)$(includedir) libdir=$(DESTDIR)$(libdir)
 
 check:
 	@$(MAKE) -C harness check

